### ISSUE TITLE
Proxy user model - create update utility for current database

### ISSUE THREAD

**Issue by htpvu** [OA-PASS/general-issues#13]
Write utility to update birkland/assets using new data and generate a new assets image with the updated model

RET 5 days

**Comment by karenhanson** [comment]
I have more details to add to this now that the data model is documented. Since the data model involves renaming of one field, deletion of another, I have discussed an approach to running an update with @birkland.  Out of that discussion, we identified 3 important pieces needed to write this utility in Java using the `java-fedora-client`:

1. The update utility will need to use both the current version of the model plus a shaded version of the old model so that it can read records using one version and write using another.
2. In order to remove the unused fields, it would be most efficient to use PUT. The Java client does not currently support this, and so I have logged issue https://github.com/OA-PASS/java-fedora-client/issues/50 for work to support this.
3. The utility will need to reliably retrieve every record in `/fcrepo/rest/submissions` in order to update each record. A paginated ElasticSearch query is unlikely to be reliable enough for this purpose, and so something like this should be used: 
https://github.com/OA-PASS/pass-authz/blob/master/pass-authz-tools/src/main/java/org/dataconservancy/pass/authz/tools/RepositoryCrawler.java. 
Here is an example of it's implementation: 
https://github.com/OA-PASS/pass-authz/blob/master/pass-authz-tools/src/main/java/org/dataconservancy/pass/authz/tools/main/PermissionsUpdater.java
